### PR TITLE
chore: Add test files for Course Repository

### PR DESCRIPTION
**Pull request by binchoi-yale**
# Description
Here is the section for descriptions

## Suggestions
- [ ] bullet point
- [ ] fdsf

### CODE REVIEW

**Comment by binchoi-yale** [comment]
@binchoi Hi check this code

File: test.txt [hunk 1]
```diff
@@ -2,3 +2,7 @@ test file
 test 1
 test 2
 test 3
+this is a test file
+test1
```
Comment (binchoi-yale) [inline test.txt:6]: this test code checks for the happy case

File: test.txt [hunk 2]
```diff
@@ -2,3 +2,7 @@ test file
...
+test2
+test3
```
Comment (binchoi-yale) [inline test.txt:8]: this tests for the edge case
